### ISSUE TITLE
Deprecate *.dpl files

### ISSUE THREAD

**Issue by ra3xdh** [ra3xdh/qucs_s#309]
It is possible to place diagrams directly on schematic without using a separate *.dpl files. I am planning to deprecate these files and change the default workflow since the next release. The DPL files will be kept only for backward compatibility. The *open data display* option in the document properties will be not active by default. The user will be invited to place diagram directly on schematic instead of display page. This will allow to distribute schematic and diagrams as one file.

Please post your vote if the separate *.dpl files should be kept. 

**TODO list:**

- [x] Change default document option
- [x] Add hint in the simulation dialog to place diagram on schematic
- [x] Reflect the workflow without DPL files in documentation
 
![Screenshot_20230921_091019](https://github.com/ra3xdh/qucs_s/assets/4920080/94695a1c-5a46-4632-99e4-b583ecf7fee4)

**Comment by cmat21** [comment]
Hi, I think it's a good idea that the open data display option in the document properties is not active by default and put all the diagrams on schematic.

**Comment by zergud** [comment]
It seems to me that there is no need to remove the functionality of dpl files.
it can be useful when combining results from different schematics
do not open by default - yes

**Comment by tomhajjar** [comment]
Keep dpl but do not open by default.

Some schematics like audio power amplifiers are so complicated there is no room for plots on the same page.

**Comment by ra3xdh** [comment]
I have added TODO list for this issue. The DPL files will be kept, but will not be created by default.

**Comment by ra3xdh** [comment]
I have changed the default option. Now the DPL file will be disabled by default in the document option. User can always enable it back if needed. I have also redesigned a simulation log window a bit. A new status log was added to make the simulation result (failure or success) better visible. I also removed the Start button, because the simulation always starts automatically and there is no need to start it twice.
![image](https://github.com/ra3xdh/qucs_s/assets/4920080/803982d8-73cf-4fac-84fa-8a23dab7fe9f)
